rule/redefines-builtin-id.go

```diff
@@ -162,16 +182,16 @@ func (w lintRedefinesBuiltinID) addFailure(node ast.Node, msg string) {
 	})
 }
 
-func (lintRedefinesBuiltinID) isBuiltIn(id string) (r bool, builtInKind string) {
-	if builtFunctions[id] {
+func (w lintRedefinesBuiltinID) isBuiltIn(id string) (r bool, builtInKind string) {
```

Comment:
```suggestion
func (w *lintRedefinesBuiltinID) isBuiltIn(id string) (r bool, builtInKind string) {
```

Reply:
@denisvmedia Thank you for your review.

Applied your suggested change 67857ce.